pkgs/applications/window-managers/hyprwm/hyprland/default.nix

```diff
@@ -114,17 +114,19 @@ stdenv.mkDerivation (finalAttrs: {
     then "debug"
     else "release";
 
+  mesonAutoFeatures = "disabled";
+
   mesonFlags = builtins.concatLists [
-    (lib.optional (!enableXWayland) "-Dxwayland=disabled")
-    (lib.optional legacyRenderer "-DLEGACY_RENDERER:STRING=true")
+    (lib.optional enableXWayland "-Dxwayland=enabled")
+    (lib.optional legacyRenderer "-Dlegacy_renderer=enabled")
     (lib.optional withSystemd "-Dsystemd=enabled")
   ];
 
   postInstall = ''
     ln -s ${wlroots}/include/wlr $dev/include/hyprland/wlroots
     ${lib.optionalString wrapRuntimeDeps ''
       wrapProgram $out/bin/Hyprland \
-        --suffix PATH : ${lib.makeBinPath [binutils pciutils]}
+        --suffix PATH : ${lib.makeBinPath [binutils pciutils stdenv.cc]}
```

Comment:
Hyprland has an open source license, it is already compiled by Hydra. And from my Nix knowledge, the compiler will be pulled in to the system as well, since it's an implicit dependency. Please correct me if I'm wrong.

Reply:
`stdenv.cc` is only needed when using plugins that need assembly capabilities (provided by udis86), so I believe it's not introducing any new dependencies.

However, in 0.35 there are some new wallpapers that take about 40MB, which may be what's eating up all that storage. I will try to make an option to disable installing them.